validate/validate.pb.go

```diff
@@ -1,14 +1,16 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
Can you change the file permissions on this back to 0644?

Reply:
Done! 👍